organizations.json

```diff
@@ -2030,8 +2030,8 @@
         "rss": "https://www.mysociety.org/feed/",
         "projects_list_url": "https://github.com/mysociety/",
         "city": "Oxford, UK",
-        "latitude": "51.7519",
-        "longitude": "1.2578",
+        "latitude": "51.62264036083571",
+        "longitude": "-0.105290718469565",
```

Comment:
While this lat/lon change certainly makes sense (especially that the previous lat/lon is in water), can you 1) update the city to match if mySociety is in London, and 2) send me a source that I can verify that mySociety is in London? Sorry to be annoying about this but I am trying to improve the rigor with which we make changes to this file.

Reply:
No problem! I am very precise when it comes to spatial, so I like it that you want to check it too.